refinery/file_store/tasks.py

```diff
@@ -27,7 +27,7 @@ class FileImportTask(celery.Task):
 
     soft_time_limit = 3600  # 1 hour
 
-    def run(self, item_uuid):
+    def run(self, item_uuid, target_name=None):
```

Comment:
If you are changing the API then it should work for all cases. Right now it only works for two out of six which is confusing and would be hard to troubleshoot.

Reply:
that's fair, i can update that

Reply:
Thanks, could you also test all these cases (manual testing is fine)? File import is a core functionality.

Reply:
yes will do